phpBB/phpbb/message/form/admin_form.php

```diff
@@ -184,6 +193,10 @@ public function submit(\messenger $messenger)
 	*/
 	public function render(\phpbb\template\template $template)
 	{
+		global $phpbb_container;
+
+		$controller_helper = $phpbb_container->get('controller.helper');
```

Comment:
Should be passed via the container

Reply:
The idea in the future is to move this method to the controller, it doesn't look good to have controller helpers in anything that is not controllers

Reply:
This pull request is small, I will try to rewrite it a little this in the future to put everything in it's place